tower-http/src/services/fs/serve_dir/tests.rs

```diff
@@ -474,7 +473,7 @@ async fn read_partial_in_bounds() {
         )));
     assert_eq!(res.headers()["content-type"], "text/markdown");
 
-    let body = hyper::body::to_bytes(res.into_body()).await.ok().unwrap();
+    let body = to_bytes(res.into_body()).await.ok().unwrap();
```

Comment:
FYI I import these as `crate::test_helpers::self` as I find that for functions like `to_bytes` it's a lot clearer when defined as `test_helpers::to_bytes` to know immediately where it comes from